docs/DeveloperGuide.md

```diff
@@ -867,14 +876,29 @@ testers are expected to do more exploratory testing.
 
 ### 7.9 Checkout a new or existing list
 
-1. tc description
+1. Checkout to an existing file with valid data and valid format.
 
-   1. Prerequisites:
+   1. Prerequisites: One JSON file with valid data and valid format in `data/` folder.
 
-   2. Test case:
+   2. Test case: `checkout FILE_NAME`
 
-2. more tc
+      Expected: The data in the file `FILE_NAME.json` located at `data/` folder is loaded into the application.
+
+2. Checkout to a non-existent file.
+
+    1. Prerequisites: No file with the name `FILE_NAME` in `data/` folder.
+
+    2. Test case: `checkout FILE_NAME`
+
+       Expected: The file `FILE_NAME.json` is created at `data/` folder and its sample data is loaded into the application.
+
+3. Checkout to an existing file with invalid data or invalid format.
+
+    1. Prerequisites: One JSON file with invalid data or invalid format in `data/` folder.
+
+    2. Test case: `checkout FILE_NAME`
 
+       Expected: The data in the file `FILE_NAME.json` located at `data/` folder is wiped.
```

Comment:
Can also specify what the app will look like? empty applicants?

Reply:
mentioned that application will not contain any applicants.